### PR TITLE
FE feat: setup template

### PR DESCRIPTION
**Pull request by bataimx**
Setup folder structures

### CODE REVIEW

**Comment by bataimx** [comment]
@TShida951 Run "npm install" before you do some smoke test, cause i've added some package support for this project

File: src/pages/Users/index.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    if(users.length === 0) {
+      GetUser().then((results) => {
+        setUsers(results.data)
+      })
+    }
+
+    return (() => {
+      // cache latest results
+    });
+  }, []);
```
Comment (TShida951) [inline src/pages/Users/index.js]: Fix warning please

> React Hook useEffect has missing dependencies: 'setUsers' and 'users.length'. Either include them or remove the dependency array  react-hooks/exhaustive-deps

File: src/components/DailyWorkLogsTable/index.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            })}
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {users.map((user, idx) =>
+            <TableRow hover tabIndex={-1} key={idx}>
+              <TableCell>{user.name}</TableCell>
+              {targetDateRange.map((date, dateIdx) => {
+                const logtime = workLogs.filter((item) => item.date === date && item.jira_id === user.jira_id );
+                const isToday = (new Date(date)).toDateString() == (new Date()).toDateString() ? classes.is_today : '';
```
Comment (TShida951) [inline src/components/DailyWorkLogsTable/index.js]: Fix warning please

> Expected '===' and instead saw '=='  eqeqeq

File: src/pages/DailyWorkLogs/index.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+export default function DailyWorkLogs() {
+  const [currentDay, setCurrentDay] = useState(getMonday(new Date()));
+  const dateArray = renderDateArray(currentDay);
+  const [targetDateRange, setTargetDateRange] = useState(dateArray);
+  const { workLogs: [workLogs, setWorkLogs],
+    users: [users, setUsers] } = useContext(KPIStoreContext);
+
+  useEffect(() => {
+    if( workLogs.length == 0 && users.length > 0 && targetDateRange.length > 0 ) {
```
Comment (TShida951) [inline src/pages/DailyWorkLogs/index.js]: Fix warning please

> Expected '===' and instead saw '=='

File: src/pages/DailyWorkLogs/index.js [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+          return {
+            jira_id: result.data.jira_id,
+            date: result.config.params.date,
+            total_time_spent: result.data.total_time_spent
+          }
+        })
+        setWorkLogs(logs)
+      });
+    }
+  }, [workLogs, targetDateRange, users])
```
Comment (TShida951) [inline src/pages/DailyWorkLogs/index.js]: Fix warning please

> React Hook useEffect has a missing dependency: 'setWorkLogs'. Either include it or remove the dependency array  react-hooks/exhaustive-deps

**Comment by TShida951** [comment]
We don't need components/DailyWorkLogs folder?

**Pull request merged**